player/lua/ytdl_hook.lua

```diff
@@ -956,6 +956,8 @@ local function run_ytdl_hook(url)
         table.insert(command, "--" .. param)
         if arg ~= "" then
             table.insert(command, arg)
+        elseif arg == "" and param == "proxy" then
```

Comment:
This can be merged with the above condition as `if arg ~= "" or param == "proxy"`